types/storybook__addon-options/storybook__addon-options-tests.ts

```diff
@@ -10,8 +10,9 @@ setOptions({
   downPanelInRight: false,
   sortStoriesByKind: false,
   hierarchySeparator: /\//,
+  hierarchyRootSeparator: /\|/
 });
 
 setOptions({
-  hierarchySeparator: '.',
```

Comment:
should we add an example with `hierarchyRootSeparator` as a string here as well @simonhn ?

Reply:
I wasnt quite sure what this second setOptions was testing - do you mean like this `  hierarchyRootSeparator: '#',`? cheers

Reply:
yes, something like that - the `hierarchyRootSeparator` is passed into a `new RegExp(...)`, so it can be a string as well, which you correctly expressed in the types, but not in the tests.

Reply:
👍